lib/UnoCore/Targets/Android/Android.uxl

```diff
@@ -221,16 +221,33 @@
 
     <ImageFile Name="@(Project.Android.Icons.LDPI:Path || '@//Assets/DefaultIcon.png')"
         TargetWidth="36" TargetName="app/src/main/res/mipmap-ldpi/icon.png" />
-    <ImageFile Name="@(Project.Android.Icons.MDPI:Path || '@//Assets/DefaultIcon.png')"
+    <ImageFile Name="@(Project.Android.Icons.MDPI:Path || '@//Assets/adaptive_icon/mipmap-mdpi/icon.png')"
```

Comment:
Are we supposed to change the existing default icon from square to round?

From my view it makes sense to keep line 224..232 as they were before, and to delete the `icon.png` variants from the commit.

Reply:
When looking at the history, round is "newer" but looking at the docs again, perhaps I need to create an option for users of round icons.

The android wizard generates the legacy icons too, so I just made use of the new one to keep the look consistent.

Reply:
Ok, I've updated the non-round with the square legacy icons that the wizard generates.

I've also included the option for setting the one round image to then turn on the round icon option: Project.Android.Icons.ANYDPI_v26_Round

Reply:
Hang a sec, I made a typo...

Reply:
I mean that the changes on line 224..232 should be reverted thus keeping `DefaultIcon.png` as default icon, like it was before. Then the `icon.png` files in your commit becomes redundant and can be deleted.

Reply:
Oh, ok, can do that, but I just realised a problem...

I added a condition to the manifest that allows a round icon to be defined when a round icon in unoproj property is defined but when trying not to define a round icon, the android.uxl copies the "icon_round.xml" which then tries to look for assets associated with round. I've tried not defining a fuse round default icon to fall back on but the <CopyFile> doesn't accept an empty string. Is there a way to do an if statement/condition in the .uxl so as to not include the round images if not define?

Reply:
Ok, sorted